Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/guide/configuration.md

## Configuring Nuxt

> We try to use the most common modules from Nuxt Community whenever it's possible. For internationalization, we are using `nuxt-i18n`, for PWA capabilities `@nuxtjs/pwa` etc. You can find a list of the Nuxt modules used in the default theme [here](theme.html#preinstalled-modules-and-libraries). Each of them is configured in a way that works best for the majority of users.

Reviewer: ```suggestion
We try to use the most common modules from Nuxt Community whenever it's possible. For example, we use the `nuxt-i18n` package for internationalization and the `@nuxtjs/pwa` package PWA capabilities. You can find a list of the Nuxt modules used in the default theme [here](theme.html#preinstalled-modules-and-libraries). Each of them is configured in a way that works best for the majority of users.
```

> ## Configuring Middleware

Reviewer: ```suggestion
## Configuring Server Middleware
```

> There are certain things related to Integration Middleware that are not tied to any specific integration, like setting custom GraphQL queries you can configure. You can read more about the Middleware and its configuration [here](/v2/advanced/server-middleware).

Reviewer: ```suggestion
You can read more about the Server Middleware and its configuration on the [Server Middleware](/v2/advanced/server-middleware) page.
```
